.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,55 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
```

Comment:
where do you use this token?

Reply:
It's advised in the default configuration of the github action, but you're right, I don't see why it would need it.  Reviewing the code, I don't see where it's used (though I'm no expert on js).

I'll experiment on my local copy and update accordingly.